---
title: System.ServiceModel.AsyncCallbackThrewException
ms.date: 03/30/2017
ms.assetid: dfc27ca7-b904-4f29-a5b6-dc702fa54a20
ms.openlocfilehash: 24cfab417e078e3d57f1b3f6fb75aeea1234afe2
ms.sourcegitcommit: 9b552addadfb57fab0b9e7852ed4f1f1b8a42f8e
ms.translationtype: MT
ms.contentlocale: ko-KR
ms.lasthandoff: 04/23/2019
ms.locfileid: "61752778"
---
# <a name="systemservicemodelasynccallbackthrewexception"></a>System.ServiceModel.AsyncCallbackThrewException
System.ServiceModel.AsyncCallbackThrewException  
  
## <a name="description"></a>설명  
 비동기 콜백에 예외가 발생했습니다.  
  
## <a name="see-also"></a>참고자료

- [추적](../../../../../docs/framework/wcf/diagnostics/tracing/index.md)
- [추적을 사용하여 애플리케이션 문제 해결](../../../../../docs/framework/wcf/diagnostics/tracing/using-tracing-to-troubleshoot-your-application.md)
- [관리 및 진단](../../../../../docs/framework/wcf/diagnostics/index.md)
